app/app.go

```diff
@@ -194,11 +195,18 @@ func Run(ctx context.Context, conf Config) (err error) {
 		return errors.Wrap(err, "private key not matching cluster manifest file")
 	}
 
+	enrRec, err := enr.New(p2pKey)
+	if err != nil {
+		return errors.Wrap(err, "creating enr record from privkey")
+	}
+
 	log.Info(ctx, "Lock file loaded",
 		z.Str("peer_name", p2p.PeerName(tcpNode.ID())),
 		z.Int("peer_index", nodeIdx.PeerIdx),
 		z.Str("cluster_hash", lockHashHex),
```

Comment:
We already log `InitialMutationHash` here albeit in short form (only the first 7 characters), do we need a new `lock_hash` field?

The `labels` mapping at line 213 will keep referencing `lockHashHex`, is this intended behavior?

Reply:
Given the labels mapping, I assumed the existing `cluster_hash` form (short form) might be used for some o11y or alerting. Therefore I preferred to not change this. Hence, the new field which logged just once.

Reply:
Makes sense.

Just for consistency sake, since the short form is called `cluster_hash`, should we rename the full field to `cluster_hash_full`?

Reply:
Agreed, done.